diff --git a/src/main.rs b/src/main.rs
index 41a1818..cd9263d 100644
--- a/src/main.rs
+++ b/src/main.rs
@@ -64,19 +64,20 @@ fn print_system_info(fields: &Fields) -> Result<(), Box<dyn std::error::Error>>
         colors,
     } = fields;
 
+
     let cyan = COLORS.cyan;
     let blue = COLORS.blue;
     let reset = COLORS.reset;
     let system_info = format!("
-    {cyan}     ▟█▖    {blue}▝█▙ ▗█▛          {user_info} ~{reset}
-    {cyan}  ▗▄▄▟██▄▄▄▄▄{blue}▝█▙█▛  {cyan}▖        {cyan}  {blue}System{reset}        {os_name}
-    {cyan}  ▀▀▀▀▀▀▀▀▀▀▀▘{blue}▝██  {cyan}▟█▖       {cyan}  {blue}Kernel{reset}        {kernel_version}
-    {blue}     ▟█▛       {blue}▝█▘{cyan}▟█▛        {cyan}  {blue}Shell{reset}         {shell}
-    {blue}▟█████▛          {cyan}▟█████▛     {cyan}  {blue}Uptime{reset}        {uptime}
-    {blue}   ▟█▛{cyan}▗█▖       {cyan}▟█▛          {cyan}  {blue}Desktop{reset}       {desktop}
-    {blue}  ▝█▛  {cyan}██▖{blue}▗▄▄▄▄▄▄▄▄▄▄▄       {cyan}  {blue}Memory{reset}        {memory_usage}
-    {blue}   ▝  {cyan}▟█▜█▖{blue}▀▀▀▀▀██▛▀▀▘       {cyan}󱥎  {blue}Storage (/){reset}   {storage}
-    {cyan}     ▟█▘ ▜█▖    {blue}▝█▛          {cyan}  {blue}Colors{reset}        {colors}\n");
+  {cyan}     ▟█▖  {blue}▝█▙ ▗█▛        {user_info} ~{reset}
+  {cyan}  ▗▄▄▟██▄▄▄{blue}▝█▙█▛  {cyan}▖      {cyan}  {blue}System{reset}   {os_name}
+  {cyan}  ▀▀▀▀▀▀▀▀▀▘{blue}▝██  {cyan}▟█▖     {cyan}  {blue}Kernel{reset}   {kernel_version}
+  {blue}     ▟█▛     {blue}▝█▘{cyan}▟█▛      {cyan}  {blue}Shell{reset}    {shell}
+  {blue}▟█████▛        {cyan}▟█████▛   {cyan}  {blue}Uptime{reset}   {uptime}
+  {blue}   ▟█▛{cyan}▗█▖     {cyan}▟█▛        {cyan}  {blue}Desktop{reset}  {desktop}
+  {blue}  ▝█▛  {cyan}██▖{blue}▗▄▄▄▄▄▄▄▄▄     {cyan}  {blue}Memory{reset}   {memory_usage}
+  {blue}   ▝  {cyan}▟█▜█▖{blue}▀▀▀██▛▀▀▘     {cyan}󱥎  {blue}Storage{reset}  {storage}
+  {cyan}     ▟█▘ ▜█▖  {blue}▝█▛        {cyan}  {blue}Colors{reset}   {colors}\n");
 
     Ok(stdout().write_all(system_info.as_bytes())?)
 }
diff --git a/src/system.rs b/src/system.rs
index aed8752..9ef0393 100644
--- a/src/system.rs
+++ b/src/system.rs
@@ -42,7 +42,7 @@ pub fn get_root_disk_usage() -> Result<String, io::Error> {
     let usage = (used_size / total_size) * 100.0;
 
     Ok(format!(
-        "{used_size:.2} GiB / {total_size:.2} GiB ({cyan}{usage:.0}%{reset})",
+        "{used_size:<4.0} / {total_size:<4.0} GiB {cyan}{usage:.0}%{reset}",
         cyan = COLORS.cyan,
         reset = COLORS.reset,
     ))
@@ -80,7 +80,7 @@ pub fn get_memory_usage() -> Result<String, io::Error> {
     let percentage_used = (used_memory / total_memory * 100.0).round() as u64;
 
     Ok(format!(
-        "{used_memory:.2} GiB / {total_memory:.2} GiB ({cyan}{percentage_used}%{reset})",
+        "{used_memory:<4.1} / {total_memory:<4.1} GiB {cyan}{percentage_used}%{reset}",
         cyan = COLORS.cyan,
         reset = COLORS.reset,
     ))
